utils/styler.py

```diff
@@ -1,12 +1,193 @@
+
+    # Set correct legend symbol rotation
+    sym = ren.sourceSymbol().clone()
+    if variable == WET_CROSS_SECTION_AREA.name:
+        sym.deleteSymbolLayer(1)
+        max_symbol_size = 1.5
+    else:
+        sym.symbolLayers()[1].setSymbolAngle(90)
+        max_symbol_size = 2.5
+    ren.updateSymbols(sym)
+
+    # Set classes and colors
+    color_ramp = color_ramp_from_data(COLOR_RAMP_OCEAN_DEEP)
+    ren.deleteAllClasses()
+    nr_classes = len(class_bounds) - 1
+    for i in range(nr_classes):
+        ren.addClassLowerUpper(
+            lower=class_bounds[i],
+            upper=class_bounds[i + 1]
+        )
+        class_middle = (class_bounds[i] + class_bounds[i + 1]) / 2
+        sym = ren.symbolForValue(class_middle).clone()
+        color_ramp_fraction = ((i + 0.5) / nr_classes)
+        color = color_ramp.color(color_ramp_fraction)
+        sym.setColor(color)
+        ren.setLegendSymbolItem(str(i), sym)
+
+    # Symbol size
+    ren.setSymbolSizes(0.1, max_symbol_size)
+
+    # Add velocity thresholds style
+    style_manager = lyr.styleManager()
+    lyr.styleManager().removeStyle('Thresholds')
+    if variable == 'u1':
+        default_style_name = style_manager.currentStyle()  # default style name depends on language settings
+        style_name = 'Thresholds'
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'velocity_thresholds.qml'
+        style = QgsMapLayerStyle()
+        style.readFromLayer(lyr)
+        style_manager.addStyle(style_name, style)
+        style_manager.setCurrentStyle(style_name)
+        (message, success) = lyr.loadNamedStyle(os.path.join(qml_path, qml_path))
+        if not success:  # if style not loaded remove it
+            style_manager.removeStyle(style_name)
+            logger.info("Styling file not succesfully loaded: {fn}".format(fn=qml_path))
+            logger.info(message)
+        style_manager.setCurrentStyle(default_style_name)
+
+    iface.layerTreeView().refreshLayerSymbology(lyr.id())
+    lyr.triggerRepaint()
+
+
+def style_animation_node_current(
+        lyr: QgsVectorLayer,
+        percentiles: List[float],
+        variable: str,
+        cells: bool = False
+):
+    """Applies styling to Animation Toolbar node layer in 'current' mode"""
+
+    # Load basic style settings from qml file
+    if cells:
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'cell_current.qml'
+    else:
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'node_current.qml'
+    lyr.loadNamedStyle(str(qml_path))
+    ren = lyr.renderer()
+
+    # Set classes
+    if variable == 's1':
+        class_attribute_str = str('coalesce(result, z_coordinate)')
+    else:
+        class_attribute_str = str('result')
+    lyr.renderer().setClassAttribute(class_attribute_str)
+    ren.deleteAllClasses()
+    nr_classes = len(percentiles) - 1
+    percentiles[0] = -9999  # to make nodes / cells are also visible when dry and z_coordinate < percentile[0]
```

Comment:
Somewhere else I saw `-9999.`, so explicitly floating point. That was in some `np.array` if I remember correctly.

Is this integer value OK?

Reply:
I didn't lead to any issues when I tested this, but I changed it anyway (into `float(-9999)`)